lib/kubernetes-deploy/kubernetes_resource/job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class Job < KubernetesResource
+    TIMEOUT = 30.seconds
```

Comment:
Is this generous enough? What if someone's pod takes several minutes to start? I'd recommend setting this closer to the timeouts on other pod managers.